README.md

```diff
@@ -17,16 +17,31 @@ originally created by Brandon Liu for Protomaps.
   - Async `mmap` (Tokio) for local files
   - Async `http` and `https` (Reqwuest + Tokio) for URLs
   - Async `s3` (Rust-S3 + Tokio) for S3-compatible buckets
+- Creating PMTile archives
 
 ## Plans & TODOs
 
 - [ ] Documentation and example code
-- [ ] Support writing and conversion to and from MBTiles + `x/y/z`
+- [ ] Support conversion to and from MBTiles + `x/y/z`
 - [ ] Support additional backends (sync `mmap` and `http` at least)
 - [ ] Support additional async styles (e.g., `async-std`)
 
 PRs welcome!
 
+## Usage examples
+
+### Writing a PMTiles file
```

Comment:
We must add a read example here before merging, or else it will get really confusing why we have a write without read

Reply:
Come to think of it - agree, this is not a blocker for merging